infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -54,6 +54,7 @@
                     + "[--base-server-threads=<base_server_threads>] "
                     + "[--log-size-quota-percentage=<max_log_size_percentage>]"
                     + "[--logunit-threads=<logunit_threads>] [--management-server-threads=<management_server_threads>]"
+                    + "[--stream-names-file=<stream_names_file>]"
```

Comment:
Could we explore a more dynamic approach for the server to become aware of the names? Since this file is static, it requires being aware of the table names ahead of time. And in the case when the mapping does not contain the stream name, since the UUID will be logged anyway, we are left with this awkward combination of both formats. Also, although I realize that this isn't the case in NSX, there are no restrictions on the format of table names, so it could contain things like new lines or special characters that make the logs difficult to read. With a UUID, this problem isn't there.

Reply:
Yes that is the long term solution. This is a best effort next best thing in its absence. 
In order for the dynamic approach we have to essentially rethink how stream names map to stream Ids. currently the solution is to use `UUID.nameUUIDFromBytes(string.getBytes());` at the client side which won't work.
Instead we need a way to ship this stream name over to the server and have the server generate the stream id.
Doing this will also allow us to generate a much more concise streamId mapping instead of burning 16 bytes.
this approach as you can see involves a lot more changes than the scope of this simple debugging fix PR.